### ISSUE TITLE
The "click color to display  hex number" behavior does not work with Voiceover turned on

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3178]
## Description
The "click color to display hex number" feature does not work when using VoiceOver/Safari/iOS 12.1.4. It is not possible to swipe to and activate the individual color segments of the SVG's used for this feature. See one of these SVG's below.

`<svg _ngcontent-c6="" class="donut" height="100%" width="100%" viewBox="0 0 48 48"><!----><path _ngcontent-c6="" class="bg-segment ng-star-inserted" d="M24,24 L24,3 A21,21 0 0,1 42.18653347947321,13.500000000000002 z" cx="24" cy="24" fill="#25333D" transform="rotate(0 24 24)"></path><path _ngcontent-c6="" class="bg-segment ng-star-inserted" d="M24,24 L24,3 A21,21 0 0,1 42.18653347947321,13.500000000000002 z" cx="24" cy="24" fill="#A6D8E7" transform="rotate(60 24 24)"></path><path _ngcontent-c6="" class="bg-segment ng-star-inserted" d="M24,24 L24,3 A21,21 0 0,1 42.18653347947321,13.500000000000002 z" cx="24" cy="24" fill="#798893" transform="rotate(120 24 24)"></path><path _ngcontent-c6="" class="bg-segment ng-star-inserted" d="M24,24 L24,3 A21,21 0 0,1 42.18653347947321,13.500000000000002 z" cx="24" cy="24" fill="#49AFD9" transform="rotate(180 24 24)"></path><path _ngcontent-c6="" class="bg-segment ng-star-inserted" d="M24,24 L24,3 A21,21 0 0,1 42.18653347947321,13.500000000000002 z" cx="24" cy="24" fill="#C1CDD4" transform="rotate(240 24 24)"></path><path _ngcontent-c6="" class="bg-segment ng-star-inserted" d="M24,24 L24,3 A21,21 0 0,1 42.18653347947321,13.500000000000002 z" cx="24" cy="24" fill="#0065AB" transform="rotate(300 24 24)"></path><g _ngcontent-c6=""><!----><path _ngcontent-c6="" class="clickable-segment ng-star-inserted" d="M24,24 L24,4 A20,20 0 0,1 41.32050807568878,14.000000000000002 z" cx="24" cy="24" fill="#25333D" transform="rotate(0 24 24)"></path><path _ngcontent-c6="" class="clickable-segment ng-star-inserted" d="M24,24 L24,4 A20,20 0 0,1 41.32050807568878,14.000000000000002 z" cx="24" cy="24" fill="#A6D8E7" transform="rotate(60 24 24)"></path><path _ngcontent-c6="" class="clickable-segment ng-star-inserted" d="M24,24 L24,4 A20,20 0 0,1 41.32050807568878,14.000000000000002 z" cx="24" cy="24" fill="#798893" transform="rotate(120 24 24)"></path><path _ngcontent-c6="" class="clickable-segment ng-star-inserted" d="M24,24 L24,4 A20,20 0 0,1 41.32050807568878,14.000000000000002 z" cx="24" cy="24" fill="#49AFD9" transform="rotate(180 24 24)"></path><path _ngcontent-c6="" class="clickable-segment ng-star-inserted" d="M24,24 L24,4 A20,20 0 0,1 41.32050807568878,14.000000000000002 z" cx="24" cy="24" fill="#C1CDD4" transform="rotate(240 24 24)"></path><path _ngcontent-c6="" class="clickable-segment ng-star-inserted" d="M24,24 L24,4 A20,20 0 0,1 41.32050807568878,14.000000000000002 z" cx="24" cy="24" fill="#0065AB" transform="rotate(300 24 24)"></path><circle _ngcontent-c6="" fill="white" cx="24" cy="24" r="12"></circle><!----><text _ngcontent-c6="" class="copy-text" dy="5.5" font-size="2.8" text-anchor="middle" x="24" y="24"> Copied! </text></g></svg>`

## Location
https://clarity.design/documentation/color

## Priority     
High

## Recommended fix    
 I recommend to hide these SVG's from screen readers with `aria-hidden="true and keeping the SVG and path elements out of the tab order. Then visibly list the colors conveyed by each SVG. This will allow sighted users who can't use a mouse to see the color values for each path in each SVG. 

`<svg aria-hidden="true" tabindex="-1"  focusable="false" _ngcontent-c6="" class="donut" height="100%" width="100%" viewBox="0 0 48 48">...</svg>`

```
<ul>
<li>#0065AB</li>
<li>#25333D</li>
<li>#A6D8E7</li>
<li>#798893</li>
<li>#49AFD9</li>
<li>#C1CDD4</li>
</ul>

```
## LOE  
Medium

**Comment by gnomeontherun** [comment]
No longer an issue or resolved in the upcoming website work. Closing as resolved.
